src/Settings.ts

```diff
@@ -133,6 +136,9 @@ export const DEFAULT_SETTINGS: ExecutorSettings = {
 	kotlinArgs: "-script",
 	kotlinFileExtension: "kts",
 	kotlinInject: "",
+	ghciPath: "",
```

Comment:
I'm not familiar with Haskell at all, so you would know this better than I: is there a "default path" for ghci? Most of the other languages have a default set (e.g. javascript has `node`, kotlin has `kotlinc`).